fuzz/fuzz.c

```diff
@@ -400,6 +400,12 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
     if (!LEN_IN_RANGE(size))
         return 0;
 
+#ifdef FONTCONFIG_SYSROOT
+#define STR_(x) #x
+#define STR(x) STR_(x)
+    setenv("FONTCONFIG_SYSROOT", STR(FONTCONFIG_SYSROOT), 1);
+#endif
```

Comment:
It would be better to do this only once during `LLVMFuzzerInitialize`.
Also i think we should not encroach on fontconfigs macro namespace here and prefix this with `ASSFUZZ_`

Reply:
done